test/test_ips.py

```diff
@@ -0,0 +1,67 @@
+import os
+import time
+import unittest
+import packet
+
+from datetime import datetime
+
+
+class TestIps(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.manager = packet.Manager(auth_token=os.environ['PACKET_AUTH_TOKEN'])
+
+        org_id = cls.manager.list_organizations()[0].id
+        cls.project = cls.manager.create_organization_project(
+            org_id=org_id,
+            name="Int-Tests-IPs_{}".format(datetime.utcnow().timestamp())
+        )
+
+        cls.ipaddresses = cls.manager\
+            .reserve_ip_address(project_id=cls.project.id,
+                                type="global_ipv4",
+                                quantity=1,
+                                facility="EWR1",
+                                details="delete me",
+                                tags=["deleteme"])
+
+        cls.device = cls.manager.create_device(
+            cls.project.id, "iptest", "baremetal_0", "ewr1", "centos_7"
+        )
+
+        while True:
+            if cls.manager.get_device(cls.device.id).state == "active":
+                break
+            time.sleep(2)
+
+        cls.manager.reserve_ip_address(project_id=cls.project.id,
+                                       type="global_ipv4",
+                                       quantity=1,
+                                       facility="ewr1")
+
+    def list_project_ips(self):
+        ips = self.manager.list_project_ips(self.project.id)
+
+        self.assertIsNotNone(ips)
+
+    def test_create_device_ip(self):
+        ip = None
+        params = {
+            "include": ["facility"]
```

Comment:
`include` param should be a comma-separated string: https://www.packet.com/developers/api/#common-parameters

If you pass it like this, it will go to the GET as
`"GET /projects/cc99d24b-f23e-43e2-867e-275918dddcf3/ips?include=['facility']`

It should be
`"GET /projects/cc99d24b-f23e-43e2-867e-275918dddcf3/ips?include=facility`

Reply:
The test has been updated.